index.html

```diff
@@ -10857,6 +10857,40 @@ <h2>A project for first-time contributions</h2>
             </div>
             <!-- ________ Marc's card END ________  -->
 
+                         <!-- ________ Hafsah's card START ________  -->
```

Comment:
Hi @Iamhafsah ! You've correctly added code making you one step closer to merging this PR. However, there is some issue with the indentation. Could you please fix the indentation so that it is consistent with the rest of the code?

If you need any help, please let us know.

Reply:
Hi @sanilborkar Thanks for observing the issue. I have resolved it, and it now uniform with the rest of the code.

Reply:
Hi. I see that you've only indented only the comment lines correctly. I was talking about your entire card. You need to indent your entire code as per the rest of the file. You can refer to the card above your code for the proper indentation.
Please let me know if you've any more questions or if you need help. Thank you!

Reply:
That was very quick! Thank you for updating the PR. Appreciate it.